Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN apt-get update && \
 
 ADD php.ini /usr/local/etc/php/php.ini
 
+# Add required php.ini directives
+RUN echo date.timezone = 'CST6CDT' >> /usr/local/etc/php/php.ini
```

Comment:
Why not just modify `php.ini` directly, since it's available within the repository?